### DISCUSSION TITLE
ModuleNotFoundError: No module named 'any module name'

### DISCUSSION THREAD

**Discussion by ostap667inbox** [bmaltais/kohya_ss#117] [Q&A]
The GUI runs, but each function gives similar errors about one of the missing modules. For example:
`Traceback (most recent call last):
File "C:\kohya_ss\networks\extract_lora_from_models.py", line 7, in <module>
import torch
ModuleNotFoundError: No module named 'torch'`
(Not only 'torch', there are errors about other modules too.)

At the same time, all necessary modules are installed. (AUTOMATIC1111 is installed and works fine).
I've already tried uninstalling and reinstalling modules more than once.

All installation steps are correct, no steps were skipped.  The installation was performed, according to the instructions, as a normal user (not an administrator).
Is there any advice you can give with my problem?

**Comment by ostap667inbox** [comment]
Problems begin to be solved as soon as you ask the question out loud :)
The question seems to be resolved. I used another terminal built into Far Manager, not PowerShell. For some reason, venv activate didn't work in it

**Comment by Pico52** [comment]
I have this problem where it complained XFormers isn't installed when it was Kohya itself that did the installation.  It's also telling me the GPU isn't set.  I'm not sure if that's related.
